sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosContainerProperties.java

```diff
@@ -178,6 +179,27 @@ public CosmosContainerProperties setChangeFeedPolicy(ChangeFeedPolicy value) {
         return this;
     }
 
+
+    /**
+     * Gets the computedProperties for this container in the Azure Cosmos DB service.
+     *
+     * @return the computedProperties.
+     */
+    public Collection<ComputedProperty> getComputedProperties() {
+        return this.documentCollection.getComputedProperties();
+    }
+
+    /**
+     * Sets the computedProperties for this container in the Azure Cosmos DB service.
```

Comment:
Is there more documentation available for this? like what ComputedProperty is, and what it does, how can customers use it, etc. Or even link to a public doc if possible